src/test/java/com/facebook/openwifirrm/optimizers/TestUtils.java

```diff
@@ -204,113 +216,128 @@ public static List<WifiScanEntry> createWifiScanList(
 		return wifiScanList;
 	}
 
-	/** Create a device state object with the given radio channel. */
-	public static State createState(int channel, int channelWidth, String bssid) {
-		return createState(channel, channelWidth, 20, 1, 20, 0, bssid);
+	/**
+	 * Create an uplink interface with one radio, to place in
+	 * {@link State#interfaces}.
+	 *
+	 * @param index index of this interface in {@link State#interfaces}
+	 * @return an uplink interface with no radios, to place in
+	 *         {@link State#interfaces}
+	 */
+	private static State.Interface createUp0StateInterface(int index) {
```

Comment:
Can you remove the `0` and `1` in these method names? It makes too little sense to someone reading it, just put the hardcoded name in the javadoc comment.